internal/database/ids.go

```diff
@@ -12,7 +11,7 @@ func (d *DBClient) GetIDsByModifiedOn(start time.Time, end time.Time) (ids []str
 	//TODO: parse name of id and modified_on fields from avro schema
 	query := fmt.Sprintf(
 		`SELECT id FROM %s WHERE modified_on > '%s' AND modified_on < '%s' ORDER BY id `,
-		d.Config.Table, start.Format(utils.TimeFormat()), end.Format(utils.TimeFormat()))
+		d.Config.Table, start.Format(time.RFC3339Nano), end.Format(time.RFC3339Nano))
```

Comment:
Me or someone knowledgeable should check the impact on the querying when the query would get a timezone.

cc @strider

Reply:
Posgresql internally stores the timestamps with timezone as UTC. Since the `modified_on` field supports TZ (https://github.com/RedHatInsights/insights-host-inventory/blob/396489360f2a9c9c68cd0840f40f9ad41f201e46/migrations/versions/ada40da60e4e_add_groups_and_hosts_groups_tables.py#L29), adding TZ info in the WHERE clause should not have any impact.